libr/core/cmd_print.c

```diff
@@ -4354,13 +4355,56 @@ static const char* bits_to_c_code_fmtstr(int bits) {
 	}
 }
 
+static void print_c_instructions(RPrint *p, ut64 addr, const ut8 *buf, int len, int ws) {
+	size_t i;
+
+	ws = R_MAX (1, R_MIN (ws, 8));
+	int bits = ws * 8;
+	const char *fmtstr = bits_to_c_code_fmtstr (bits);
+	len /= ws;
+
+	p->cb_printf ("#define _BUFFER_SIZE %d\n", len);
+	p->cb_printf ("const uint%d_t buffer[_BUFFER_SIZE] = {\n  ", bits);
+	p->interrupt = false;
+
+	int left = 0;
+	const int orig_align = p->coreb.cfggeti (p->coreb.core, "asm.cmt.col") - 40;
+	int align = orig_align;
+	char *instr = NULL;
+	for (i = 0; !p->interrupt && i < len; i++) {
+		if (left == 0) {
+			ut64 at = addr + i;
+			char *is = p->coreb.cmdstrf (p->coreb.core, "ao @ 0x%08"PFMT64x"~^size[1]", at);
```

Comment:
1) no cmdstrf because you can use directly `r_core_cmdf` afaik.
2) just use the disasm api to get the size of the instruction.

Reply:
no, i cant
this is r_utl, not r_core
no i cant sue the disasm api either